appveyor.yml

```diff
@@ -15,24 +15,21 @@ environment:
     CODECOV_ENV: PYTHON_VERSION, PLATFORM
 
   matrix:
-    - PYTHON_VERSION: 2.7 
+    - PYTHON_VERSION: 3.8
       BUILD_OPTS: --xplat
-    - PYTHON_VERSION: 3.5
+    - PYTHON_VERSION: 3.7
       BUILD_OPTS: --xplat
     - PYTHON_VERSION: 3.6
       BUILD_OPTS: --xplat
-    - PYTHON_VERSION: 3.7
-      BUILD_OPTS: --xplat
-    - PYTHON_VERSION: 2.7
     - PYTHON_VERSION: 3.5
-    - PYTHON_VERSION: 3.6
+      BUILD_OPTS: --xplat
```

Comment:
NIT: can you minimize changes here and in .travis.yml if possible?

Reply:
I can move them to a single separate commit, but the changes are important for my sanity. I don't want to see old, unsupported versions fly by in CI while I wait 15min for the actually interesting thing. The only change here is that I reordered the build matrix to list the versions in descending order.

Reply:
Actually, they are already in different commits (adding Py3.8 and reordering).